### ISSUE TITLE
bug:

### ISSUE THREAD

**Issue by jotalp** [withfig/fig#2731]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.6.9 (21G726)

### Expected behaviour

I expected to be able to sign in with github... this is what I got. 
<img width="1440" alt="Screen Shot 2023-09-13 at 12 50 54 PM" src="https://github.com/withfig/fig/assets/28308860/dfc4b30a-e615-473b-8073-0fdae729e3f3">

### Actual behaviour

<img width="1440" alt="Screen Shot 2023-09-13 at 12 50 54 PM" src="https://github.com/withfig/fig/assets/28308860/444e9dd3-6d06-4097-804b-99b26326b377">

### Environment

```yaml
fig-details:
  - 2.16.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  - cores: 4
  - mem: 16.00 GB
os-info:
  - macOS 12.6.9 (21G726)
environment:
  - shell: /Applications/Fig.app/Contents/MacOS/fig_desktop
  - terminal: <unknown>
  - cwd: /
  - exe-path: /Applications/Fig.app/Contents/MacOS/fig-darwin-universal
  - install-method: unknown
  - env-vars:
    - __CFBundleIdentifier: com.mschrage.fig
    - PATH: /usr/bin:/bin:/usr/sbin:/sbin
    - SHELL: /bin/zsh
```

**Comment by mschrage** [comment]
Closing as a duplicate of #2705.

tl;dr. Fig is joining AWS. We are working with the security team to reopen logins.
